velox/docs/monthly-updates/july-2022.rst

```diff
@@ -0,0 +1,58 @@
+****************
+July 2022 Update
+****************
+
+Documentation
+=============
+
+* Add documentation for Presto related :doc:`/functions/json`.
+
+Core Library
+============
+
+* Add `ApproxMostFrequentStreamSummary` that approximately computes the top frequent values from a large stream.
+* Add support for cast from a JSON type value to ARRAYS or MAPS of JSON type values.
+* Add Parquet writer implemented over Apache Arrow Parquet writer.
+* Add NegatedBytesRange filter.
```

Comment:
Perhaps, clarify this a bit.

`* Add NegatedBytesRange filter to optimize NOT IN ('a', 'b', 'c') filters on string columns pushed down into DWRF reader.`